src/apps/mahalend/arbitrum/mahalend.claimable.contract-position-fetcher.ts

```diff
@@ -0,0 +1,14 @@
+import { PositionTemplate } from '~app-toolkit/decorators/position-template.decorator';
+
+import { AaveV2ClaimablePositionFetcher } from '../common/mahalend.claimable.contract-position-fetcher';
+
+@PositionTemplate()
+export class ArbitrumAaveV2ClaimableContractPositionFetcher extends AaveV2ClaimablePositionFetcher {
```

Comment:
You will need to rename the class so it contains the name of the app intestead of `AaveV2`, otherwise it be rendered under the mahalend app. Have a look at the `@PositionTemplate()`'s code

```suggestion
export class ArbitrumMahalendClaimableContractPositionFetcher extends AaveV2ClaimablePositionFetcher {
```